src/audio/Kconfig

```diff
@@ -72,6 +72,13 @@ config COMP_SRC
 	help
 	  Select for SRC component
 
+config NOTIFY_HOST_ON_XRUN
+	bool "Notify the host via IPC that a XRUN/Glitch has occurred"
+	default n
+	help
+	  Select for handling DMA xruns and reporting them as errors.
+	  Otherwise, only warnings are emitted.
+
```

Comment:
I won't block on this, but if we have the Kconfig, I'd do:
- put it at the end of audio/Kconfig after WRAP_ACTUAL_POSITION (now it's in middle of SRC related options)
- name it DAI_REPORT_XRUNS
- description: "Report DAI overrun/underruns to pipeline"

This would better reflect what this does.